changelog.md

```diff
@@ -27,7 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
```

Comment:
you did not add the plugin core update haha

Reply:
done